<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="ActionNotSupportedException.xml" source-language="en-US" target-language="it-IT">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac578e3df2962e97494f675e54c73874da310d7557f.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">78e3df2962e97494f675e54c73874da310d7557f</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.ServiceModel.ActionNotSupportedException">
          <source>This exception is typically thrown on the client when the action related to the operation invoked does not match any action of operations in the server.</source>
          <target state="translated">Questa eccezione viene solitamente generata nel client quando l'azione correlata all'operazione richiamata non corrisponde ad alcuna azione di operazioni nel server.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ServiceModel.ActionNotSupportedException">
          <source>The (partial) match failure on the service creates a <ph id="ph1">`wsa:ActionNotSupported`</ph> fault on the wire that is turned into a <ph id="ph2">&lt;xref:System.ServiceModel.ActionNotSupportedException&gt;</ph> on the client.</source>
          <target state="translated">L'errore di corrispondenza (parziale) nel servizio crea un errore <ph id="ph1">`wsa:ActionNotSupported`</ph> in transito che viene trasformato in una <ph id="ph2">&lt;xref:System.ServiceModel.ActionNotSupportedException&gt;</ph> nel client.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ServiceModel.ActionNotSupportedException">
          <source>This exception derives from <ph id="ph1">&lt;xref:System.ServiceModel.CommunicationException&gt;</ph> that represents a class of recoverable errors that may be thrown during communication between endpoints and that robust <ph id="ph2">[!INCLUDE[indigo1](~/includes/indigo1-md.md)]</ph> client and service applications are expected to handle.</source>
          <target state="translated">Questa eccezione deriva dall'eccezione <ph id="ph1">&lt;xref:System.ServiceModel.CommunicationException&gt;</ph> che rappresenta una classe di errori reversibili che possono essere generati durante la comunicazione tra endpoint e che si prevede siano gestiti da applicazioni <ph id="ph2">[!INCLUDE[indigo1](~/includes/indigo1-md.md)]</ph> affidabili sia lato server sia lato client.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ServiceModel.ActionNotSupportedException">
          <source>To prevent a more generic handler for <ph id="ph1">&lt;xref:System.ServiceModel.CommunicationException&gt;</ph> from catching the more specific <ph id="ph2">&lt;xref:System.ServiceModel.ActionNotSupportedException&gt;</ph>, catch this exception prior to handling <ph id="ph3">&lt;xref:System.ServiceModel.CommunicationException&gt;</ph>.</source>
          <target state="translated">Per evitare che un gestore più generico di <ph id="ph1">&lt;xref:System.ServiceModel.CommunicationException&gt;</ph> rilevi l'eccezione <ph id="ph2">&lt;xref:System.ServiceModel.ActionNotSupportedException&gt;</ph> più specifica, rilevare questa eccezione prima di gestire le eccezioni <ph id="ph3">&lt;xref:System.ServiceModel.CommunicationException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="T:System.ServiceModel.ActionNotSupportedException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ServiceModel.ActionNotSupportedException" /&gt;</ph> class.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.ServiceModel.ActionNotSupportedException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ServiceModel.ActionNotSupportedException">
          <source>If you want to pass an error message to the user, use the <ph id="ph1">&lt;xref:System.ServiceModel.ActionNotSupportedException.%23ctor%28System.String%29&gt;</ph> constructor.</source>
          <target state="translated">Se si desidera passare un messaggio di errore all'utente, utilizzare il costruttore <ph id="ph1">&lt;xref:System.ServiceModel.ActionNotSupportedException.%23ctor%28System.String%29&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ServiceModel.ActionNotSupportedException">
          <source>If you want to pass an error message and a reference to the inner exception that is the cause of the exception to the user, use the <ph id="ph1">&lt;xref:System.ServiceModel.ActionNotSupportedException.%23ctor%28System.String%2CSystem.Exception%29&gt;</ph> constructor.</source>
          <target state="translated">Se si desidera passare un messaggio di errore e un riferimento all'eccezione interna che ha generato l'eccezione, utilizzare il costruttore <ph id="ph1">&lt;xref:System.ServiceModel.ActionNotSupportedException.%23ctor%28System.String%2CSystem.Exception%29&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ServiceModel.ActionNotSupportedException">
          <source>If you want to pass serialization information and streaming context, use the <ph id="ph1">&lt;xref:System.ServiceModel.ActionNotSupportedException.%23ctor%28System.Runtime.Serialization.SerializationInfo%2CSystem.Runtime.Serialization.StreamingContext%29&gt;</ph> constructor.</source>
          <target state="translated">Se si desidera passare le informazioni sulla serializzazione e il contesto di flusso, utilizzare il costruttore <ph id="ph1">&lt;xref:System.ServiceModel.ActionNotSupportedException.%23ctor%28System.Runtime.Serialization.SerializationInfo%2CSystem.Runtime.Serialization.StreamingContext%29&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ServiceModel.ActionNotSupportedException" /&gt;</ph> class.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.ServiceModel.ActionNotSupportedException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.String)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Messaggio di errore in cui viene indicato il motivo dell'eccezione.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ServiceModel.ActionNotSupportedException" /&gt;</ph> class with a specified error message.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.ServiceModel.ActionNotSupportedException" /&gt;</ph> con un messaggio di errore specificato.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.String)">
          <source>Use this constructor when you want to pass an error message to the user.</source>
          <target state="translated">Utilizzare questo costruttore quando si desidera passare all'utente un messaggio di errore.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.String)">
          <source>The content of the message parameter should be understandable to the intended users.</source>
          <target state="translated">Il contenuto del messaggio deve essere comprensibile agli utenti previsti.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that contains all the data required to serialize the exception.</source>
          <target state="translated">Oggetto della classe <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> contenente tutti i dati necessari per serializzare l'eccezione.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that specifies the source and destination of the stream.</source>
          <target state="translated">Struttura <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> che definisce l'origine e la destinazione del flusso.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ServiceModel.ActionNotSupportedException" /&gt;</ph> class with serialization information and streaming context specified.</source>
          <target state="translated">Consente di inizializzare una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.ServiceModel.ActionNotSupportedException" /&gt;</ph> con le informazioni di serializzazione e il contesto di flusso specificati.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is called during deserialization to reconstruct the exception object transmitted over a stream.</source>
          <target state="translated">Questo costruttore viene chiamato nel corso della deserializzazione per ricostruire l'oggetto eccezione trasmesso in un flusso.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Messaggio di errore in cui viene indicato il motivo dell'eccezione.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that caused the current exception to be thrown.</source>
          <target state="translated">Eccezione <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> che ha causato la generazione dell'eccezione corrente.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ServiceModel.ActionNotSupportedException" /&gt;</ph> class with a specified error message and a reference to the inner exception that is the cause of the exception.</source>
          <target state="translated">Consente di inizializzare una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.ServiceModel.ActionNotSupportedException" /&gt;</ph> con il messaggio di errore specificato e un riferimento all'eccezione interna che ha generato l'eccezione.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.String,System.Exception)">
          <source>Use this constructor when you want to pass an error message and information about an inner exception to the user.</source>
          <target state="translated">Utilizzare questo costruttore quando si desidera passare all'utente un messaggio di errore e le informazioni su un'eccezione interna.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.String,System.Exception)">
          <source>The content of the message parameter should be understandable to the intended users.</source>
          <target state="translated">Il contenuto del messaggio deve essere comprensibile agli utenti previsti.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Un'eccezione generata come risultato diretto di un'eccezione precedente può includere un riferimento all'eccezione precedente nella proprietà <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ServiceModel.ActionNotSupportedException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor or a <ph id="ph2">`null`</ph> reference (<ph id="ph3">`Nothing`</ph> in Visual Basic) if the <ph id="ph4">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">La proprietà <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> restituisce lo stesso valore passato al costruttore o un riferimento <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph> in Visual Basic) se la proprietà <ph id="ph4">&lt;xref:System.Exception.InnerException%2A&gt;</ph> non fornisce al costruttore il valore dell'eccezione interna.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>